addons/crm/models/crm_lead.py

```diff
@@ -1108,6 +1108,27 @@ def _merge_opportunity_history(self, opportunities):
                 })
         return True
 
+    def _merge_opportunity_followers(self, opportunities):
+        """ Move followers of given opportunities to the current one `self`
+
+        :param opportunities: see ``merge_dependences``
+        """
+        self.ensure_one()
+
+        # return followers of opportunity
+        def _get_followers(opportunity_id):
+            return self.env['mail.followers'].search([('res_model', '=', self._name), ('res_id', '=', opportunity_id)])
+        # make sure not to have duplicated followers
+        followers_partner_ids = _get_followers(self.id).partner_id.ids
```

Comment:
the similar method below use `_merge_opportunity_attachments` use records and not IDs, I think you could do the same (and remove .ids here, and other part where followers_partner_ids is used)